CorsixTH/Lua/dialogs/resizables/machine_menu.lua

```diff
@@ -0,0 +1,341 @@
+      :makeButton(0, 0, 40, 20, nil, self.setSortTU)
+      :setTooltip(_S.tooltip.machine_menu.header.times_used .. " " .. _S.tooltip.machine_menu.sort)
+
+    for i = 1, rows, 1 do
+      local smoke_panel = self:addBevelPanel(window_margin, y+row_height, 20, row_height, col_smoke_button)
+      local smoke_button = smoke_panel:makeButton(0, 0, 20, row_height, nil, smoke_factory(i))
+        :setTooltip(_S.tooltip.machine_menu.smoking)
+      local assigned_panel = self:addBevelPanel(window_margin+20, y+row_height, 20, row_height, col_assigned_button)
+      local assigned_button = assigned_panel:makeButton(0, 0, 20, row_height, nil, assigned_factory(i))
+        :setTooltip(_S.tooltip.machine_menu.assigned)
+      local task_panel = self:addBevelPanel(window_margin+40, y+row_height, self.width - 190 - 40 - window_margin, row_height, col_bg)
+      local task_button = task_panel:makeButton(0, 0, self.width - 190 - 40 - window_margin, row_height, nil, task_factory(i))
+        :setTooltip(_S.tooltip.machine_menu.machine)
+      local remaining_strength_panel = self:addBevelPanel(280, y+row_height, 40, row_height, col_shadow)
+        :setTooltip(_S.tooltip.machine_menu.remaining_strength)
+      local strength_panel = self:addBevelPanel(320, y+row_height, 40, row_height, col_shadow)
+        :setTooltip(_S.tooltip.machine_menu.total_strength)
+      local percentage_panel = self:addBevelPanel(360, y+row_height, 40, row_height, col_shadow)
+        :setTooltip(_S.tooltip.machine_menu.ratio)
+      local usage_panel = self:addBevelPanel(400, y+row_height, 40, row_height, col_shadow)
+        :setTooltip(_S.tooltip.machine_menu.times_used)
+      table.insert(self.list_table, {
+        smoke_header = smoke_header,
+        assigned_header = assigned_header,
+        name_header = name_header,
+        remaining_strength_panel_header = remaining_strength_panel_header,
+        strength_panel_header = strength_panel_header,
+        ratio_panel_header = ratio_panel_header,
+        timesUsed_panel_header = timesUsed_panel_header,
+        assigned_panel = assigned_panel,
+        assigned_button = assigned_button,
+        smoke_panel = smoke_panel,
+        smoke_button = smoke_button,
+        task_panel = task_panel,
+        task_button = task_button,
+        remaining_strength_panel = remaining_strength_panel,
+        strength_panel = strength_panel,
+        percentage_panel = percentage_panel,
+        usage_panel = usage_panel
+      })
+      y = y + row_height
+    end
+    self.summary_panel = self:addColourPanel(50, y + 30, self.width - 50 - 40 - window_margin, 20,
+                                             col_bg.red, col_bg.green, col_bg.blue)
+    self.scrollbar = self:addColourPanel(self.width - window_margin - 20, window_margin+20, 20, row_height * rows,
+                                         col_shadow.red, col_shadow.green, col_shadow.blue)
+                         :makeScrollbar(col_bg, callback, 1, 1, 10, 1)
```

Comment:
```suggestion
    self.summary_panel = self:addColourPanel(50, y + 30, self.width - 50 - 40 - window_margin, 20,
                             col_bg.red, col_bg.green, col_bg.blue)
    self.scrollbar = self:addColourPanel(self.width - window_margin - 20, window_margin+20, 20, row_height * rows,
                         col_shadow.red, col_shadow.green, col_shadow.blue)
                       :makeScrollbar(col_bg, callback, 1, 1, 10, 1)
```
Indenting for windows is a little odd; but this one aligned in a way that just feels messy. I've moved your runoff line for 174 into 175 to only be 4 spaces indented from the `self`; same witgh 176 and 177.
Line 178 is only 2 spaces indented; as it appears a special case for window code that those colon calls don't follow standard 4 line indentation (see line 180)